### PR TITLE
[MDS-4745] Core web Version Fixes - Part 1

### PR DESCRIPTION
**Pull request by vyasworks**
## Objective 

[MDS-4745](https://bcmines.atlassian.net/browse/MDS-4745)

### CODE REVIEW

**Comment by vyasworks** [comment]
Testing Minespace Upgrades - 

`npm outdated` shows that the package lock, already has multiple packages with minor versions changed, bumping them up a bit more. 

```
@ant-design/icons                          4.6.4    4.7.0    4.7.0  @mds/minespace-web
@babel/core                               7.15.0   7.19.3   7.19.3  @mds/minespace-web
@babel/plugin-proposal-class-properties   7.14.5   7.18.6   7.18.6  @mds/minespace-web
@babel/plugin-transform-runtime           7.15.0   7.19.1   7.19.1  @mds/minespace-web
@babel/preset-env                         7.15.0   7.19.4   7.19.4  @mds/minespace-web
@babel/preset-react                       7.14.5   7.18.6   7.18.6  @mds/minespace-web
@babel/runtime                            7.15.3   7.19.4   7.19.4  @mds/minespace-web
@babel/runtime-corejs3                    7.15.3   7.19.4   7.19.4  @mds/minespace-web
autoprefixer                               9.8.6    9.8.8  10.4.12  @mds/minespace-web
axios                                     0.21.1   0.21.4    1.1.3  @mds/minespace-web
core-js                                   3.16.2   3.25.5   3.25.5  @mds/minespace-web
eslint-plugin-import                      2.24.0   2.26.0   2.26.0  @mds/minespace-web
eslint-plugin-jest                       22.21.0  22.21.0   27.1.3  @mds/minespace-web
eslint-plugin-jsx-a11y                     6.4.1    6.6.1    6.6.1  @mds/minespace-web
eslint-plugin-react                       7.24.0  7.31.10  7.31.10  @mds/minespace-web
filepond-plugin-file-validate-size         2.2.4    2.2.7    2.2.7  @mds/minespace-web
filepond-plugin-file-validate-type         1.2.6    1.2.8    1.2.8  @mds/minespace-web
leaflet                                    1.7.1    1.9.2    1.9.2  @mds/minespace-web
leaflet.markercluster                      1.5.0    1.5.3    1.5.3  @mds/minespace-web
moment                                    2.29.1   2.29.4   2.29.4  @mds/minespace-web
prop-types                                15.7.2   15.8.1   15.8.1  @mds/minespace-web
react-filepond                             7.1.1    7.1.2    7.1.2  @mds/minespace-web
react-redux                                7.2.4    7.2.9    8.0.4  @mds/minespace-web
react-redux-loading-bar                    5.0.2    5.0.4    5.0.4  @mds/minespace-web
react-router-dom                           5.2.0    5.3.4    6.4.2  @mds/minespace-web
redux                                      4.1.1    4.2.0    4.2.0  @mds/minespace-web
redux-form                                 8.3.7    8.3.8    8.3.8  @mds/minespace-web
redux-form-input-masks                     2.0.1    2.0.2    2.0.2  @mds/minespace-web
redux-thunk                                2.3.0    2.4.1    2.4.1  @mds/minespace-web
regenerator-runtime                       0.13.9  0.13.10  0.13.10  @mds/minespace-web
reselect                                   4.0.0    4.1.6    4.1.6  @mds/minespace-web
```

Same Story for core-web


```
react                                    16.13.1  16.14.0   18.2.0  @mds/core-web
react-dom                                16.13.1  16.14.0   18.2.0  @mds/core-web
react-filepond                             7.1.0    7.1.2    7.1.2  @mds/core-web
react-hot-loader                         4.12.21   4.13.0   4.13.0  @mds/core-web
react-redux                                7.2.1    7.2.9    8.0.4  @mds/core-web
react-redux-loading-bar                    5.0.0    5.0.4    5.0.4  @mds/core-web
react-responsive                           8.1.0    8.2.0    9.0.0  @mds/core-web
react-responsive-carousel                  3.2.9   3.2.23   3.2.23  @mds/core-web
react-router-dom                           5.2.0    5.3.4    6.4.2  @mds/core-web
react-scroll                               1.8.1    1.8.7    1.8.7  @mds/core-web
reactour                                  1.18.0   1.18.7   1.18.7  @mds/core-web
redux                                      4.0.5    4.2.0    4.2.0  @mds/core-web
redux-form                                 8.3.6    8.3.8    8.3.8  @mds/core-web
redux-form-input-masks                     2.0.1    2.0.2    2.0.2  @mds/core-web
redux-thunk                                2.3.0    2.4.1    2.4.1  @mds/core-web
regenerator-runtime                       0.13.7  0.13.10  0.13.10  @mds/core-web
reselect                                   4.0.0    4.1.6    4.1.6  @mds/core-web
filepond-plugin-file-encode                2.1.9   2.1.10   2.1.10  @mds/core-web
filepond-plugin-file-validate-size         2.2.1    2.2.7    2.2.7  @mds/core-web
filepond-plugin-file-validate-type         1.2.5    1.2.8    1.2.8  @mds/core-web
filepond-plugin-get-file                   1.0.6    1.0.7    1.0.7  @mds/core-web
filepond-plugin-image-exif-orientation     1.0.9   1.0.11   1.0.11  @mds/core-web
filepond-plugin-image-preview              4.6.4   4.6.11   4.6.11  @mds/core-web
```

very few changes on patch versions only or minor versions, should not affect the app

**Review by simensma-fresh — DISMISSED** [review]
Looks good to me! 🚀

File: services/core-web/src/components/Forms/incidents/IncidentForm.js [hunk 1]
```diff
@@ -777,7 +777,7 @@ const renderEditSaveControls = (props, isEditMode, isNewIncident) => (
   </div>
 );
 
-const IncidentForm = (props) => {
+export const IncidentForm = (props) => {
```
Comment (matbusby-fw) [inline services/core-web/src/components/Forms/incidents/IncidentForm.js:780]: Was the import in the test not working properly with the default export?

**Review by matbusby-fw — DISMISSED** [review]
Looks great!  Just curious on the need to change those exports/imports.

File: services/core-web/src/components/Forms/incidents/IncidentForm.js (hunk 1 above)
Comment (vyasworks) [reply services/core-web/src/components/Forms/incidents/IncidentForm.js:780]: They din't seem to work properly after some upgrades, which was odd.

Comment (matbusby-fw) [reply services/core-web/src/components/Forms/incidents/IncidentForm.js:780]: Odd.  I've been converting some elements from class based components to functional ones and have noticed these test imports get a little finicky, but I haven't seen it require this yet.  🤷

**Review by simensma-fresh — APPROVED** [review]

**Pull request merged**